pkg/pipelines/secrets/secrets.go

```diff
@@ -97,9 +98,9 @@ func getClusterPublicKey() (*rsa.PublicKey, error) {
 }
 
 // Returns a reader of public key from sealed-secrets-controller
-func openCertCluster(c clientv1.CoreV1Interface) (io.ReadCloser, error) {
+func openCertCluster(c clientv1.CoreV1Interface, ns string) (io.ReadCloser, error) {
 	f, err := c.
-		Services("kube-system").
+		Services(ns).
```

Comment:
Should we let user to specify SS controller name via flags, too?   If SS controller is installed by yaml like https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.11.0/controller.yaml, the SS controller name is "sealed-secrets-controller".

Reply:
I pondered this, but we're making this change to allow for the operator.

I don't feel strongly one way or another (it feels like _yet another command line parameter_ )

I'm happy to include the option, and happy not to.

Reply:
One benefit of effectively requiring the operator, we know what versions we need to support :-)

Reply:
It's fine without it (configurable controller name).